Jint/Engine.cs

```diff
@@ -239,6 +239,12 @@ public Engine(Action<Engine, Options> options)
             ClrTypeConverter = new DefaultTypeConverter(this);
         }
 
+        public void SetMemberAccessor(Type type, string name, Func<Engine, object, PropertyDescriptor> accessor)
```

Comment:
Is there a need to change behavior runtime or would it make sense to do these via options when initializing engine?